datalad/interface/common_opts.py

```diff
@@ -269,6 +269,11 @@
     See https://git-annex.branchable.com/git-annex-groupwanted/ for more information""",
     constraints=EnsureStr() | EnsureNone())
 
+annex_ignore_opt = Parameter(
+    args=("--annex-ignore",),
+    metavar='BOOL',
+    doc="Whether or not a remote shall be ignored or not",
+    constraints=EnsureChoice(None, True, False, 'true', 'false'))
```

Comment:
This is less general than `EnsureBool() | EnsureNone()`, but the resulting composite constraint looses the ability to generate a meaningful help. Although it would be =relatively simple and useful to finally fix this (out of scope here). This this implementation is (sadly) optimal.